apps/sms/views/shared/js/localization_helper.js

```diff
@@ -67,7 +69,7 @@
     init: function () {
       // This will be called during startup, and every time the language is
       // changed
-      navigator.mozL10n.ready(function localized() {
+      document.addEventListener('supportedlanguageschange', function() {
```

Comment:
In https://bugzilla.mozilla.org/show_bug.cgi?id=1206656 I removed the `supportedlanguageschange` event which didn't have a flag equivalent which would indicated that the event already fired.  Instead, the recommended way will be to use `document.l10n.ready` and the `DOMLocalized` event which also have the benefit of guaranteeing that you're working with a localized DOM.